Bulk edit of role checkboxes for requests

viaccoz opened the pull request:
Add the ability to select/unselect the checkboxes of role assigments in requests by right-clicking on the role column.

viaccoz commented:
Concretely, you can select several requests, right-click on the role and you'll be able to select/unselect all the checkboxes at once:
![image](https://user-images.githubusercontent.com/42694303/72624532-6baa3700-3947-11ea-9393-38bfe75386d8.png)

benstew commented on AuthMatrix.py:
```diff
@@ -385,6 +398,12 @@ def actionPerformed(self,e):
         roleRemoveFromMessageTable = JMenuItem("Remove Role")
         roleRemoveFromMessageTable.addActionListener(actionRemoveColumn("m"))
         messageHeaderPopup.add(roleRemoveFromMessageTable)
+        enableToggle = JMenuItem("Select checkboxes")
```
Also, as I look at it more, what are your thoughts on including the word "Bulk" within the menu name to provide a user with additional context? For example, `"Bulk Select Checkboxes"`. It seems like that might add a little extra clarity but curious what you think.

viaccoz commented:
@benstew I updated the version and changed the menu items to add clarity. It's a good idea to include "bulk" in the names, so it's clearer to the user.
Thanks for the awesome work done on AuthMatrix, it's a great help during the assessments! :)
